CONTRIBUTING.md

```diff
@@ -0,0 +1,177 @@
+
+* [Issues](#issues): You are a regular user, and want to report a bug, ask for a new feature or for
+  an existing feature to be improved.
+
+* [Translations](#translations): You speak English and you're fluent in another language, and want
+  to help us reach more users around the globe.
+
+* [Hosting an instance](#hosting-an-instance): You're a sysadmin and have some time and server
+  ressources to spare, with some prior experience with invidious.
+
+* [Testing](#testing): You're an advanced user, who is already running their own instance, is not
+  worried of compiling Invidious from source, and wants to help us test bug fixes or new features.
+
+* [Development](#development): You're a developer and want to contribute code, or help with
+  reviewing the code written by others.
+
+
+
+## Issues
+
+We use the [Github issue tracker](https://github.com/iv-org/invidious/issues) to track and manage
+bug reports, feature requests and improvements.
+
+**Note: Before opening any kind of issue, make sure to search on the tracker
+with various keywords to verify that no other similar issue have already been
+opened (and/or closed).**
+
+In order for everyone to be able to understand eachother, all exchanges are done in English.
+You can obviously use a translator if needed.
+
+Please be polite and respectful in your exchanges. Remember that we're volunteers providing that
+service for free. We don't have the time nor energy to deal with bad manners.
+
+
+### Bug reports
+
+The most common case is that you ended up on a page saying "you encountered a
+bug in Invidious" while you were browsing a channel or watching a video.
+
+In that case, you should open a "bug report". Please include as many details as possible, so
+that we can easily reproduce your problem on our side.
+
+Before opening your issue, **make sure to test a few other instances**, just to check that the
+problem you're facing is not temporary (E.g an overloaded instance, a network outage, etc.) or
+caused by configuration error on your side.
+
+If a bug report already exists for your problem, you can add a comment with more details, but make
+sure that it's useful and adds value to the discussion. If 20 people did that already, it might not
+be relevant to post a new comment.
+
+Here is a non-exhaustive list of details that will help us:
+
+  * **A clear and concise list of steps to reproduce the problem**
+  * A link to the page where the bug happened
+  * Browser/OS version, device type (mobile, desktop, etc..)
+  * Are you logged in?
+  * If the bug is caused by an external file (ex: when importing subscriptions), try to include
+    it too ([Get in touch](https://invidious.io/contact/) with us if you want to share these files privately).
+  * If you're hosting your own instance, include relevant config file(s).\
+    **Make sure to redact secrets like your DB password and HMAC key first!!**
+
+
+**Note: Security-related issues should be reported by e-mail at
```

Comment:
I think this is fine for now but in the future we should probably be pointing this to a proper security policy at `SECURITY.md`